go.mod

```diff
@@ -14,3 +14,5 @@ require (
 	golang.org/x/sys v0.0.0-20190712062909-fae7ac547cb7 // indirect
 	golang.org/x/tools v0.0.0-20190724185037-8aa4eac1a7c1 // indirect
 )
+
+go 1.13
```

Comment:
did you use anything here which requires go 1.13? I'd like to avoid shutting out users of earlier versions of Go - since there's a compile time dependency on our runtime package.

Reply:
Oh my bad. I must have committed it by mistake - `go mod` is quite intrusive. No reason to add a requirement on 1.13.